java/code/src/com/redhat/rhn/frontend/xmlrpc/system/SystemHandler.java

```diff
@@ -678,6 +679,26 @@ public long scheduleChangeChannels(User loggedInUser, Integer serverId, String b
         return dr.toArray();
     }
 
+    /**
+     * Returns a list of empty system profiles visible to user (created by createSystemProfile).
+     *
+     * @param loggedInUser - the user
+     * @return array of empty system profiles
+     *
+     * @xmlrpc.doc Returns a list of empty system profiles visible to user (created by the createSystemProfile method).
+     * @xmlrpc.param #session_key()
+     *
+     * @xmlrpc.returntype
+     *          #array()
+     *              $SystemOverviewSerializer
```

Comment:
I think you understood me wrong :-)
This was just an example. I think the SystemOverviewSerializer will not work as you created a BootstrapOverview object which has extra info.

I think you need to write a BootstrapOverviewSerializer which return the values you really use.

Reply:
Technically the `SystemOverviewSerializer` works, but you're right - the empty profiles have some extra info (mainly the HW address). I think it's absolutely necessary to display it in the output.